macOS/DuckDuckGo/VisualRefresh/ThemeName.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ThemeName.swift
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+enum ThemeName: String, CaseIterable {
```

Comment:
This could be a typealias. I’m opting for having the macOS codebase to explicitly own the supported Palettes (rather than deferring to DRK).

That being said, feedback more than welcomed. Thanks in advance!

```
typealias ThemeName = DesignResourcesKit.ColorPalette
```

Reply:
A typealias is a good idea 👍🏼 . About having both DRK and macOS defined palettes, I’m not oppossed, as I mentioned in a previous comment, in the future we should try to get everything from one source.